scripts/copy_from_private_repo.py

```diff
@@ -39,6 +39,8 @@
 FILE_PATTERNS.append('meta.properties') # TODO: create meta.edn plugins for extensions
 FILE_PATTERNS.append('build.xml') # TODO: make a strategy here
 
+FILE_PATTERNS.append('platform.sdks.json')
```

Comment:
Probably not strictly necessary, and will always likely cause a conflict when merging from public to private, as the file will change version numbers a lot.